testing/scripts/seldon_e2e_utils.py

```diff
@@ -418,3 +436,49 @@ def clean_string(string):
     string = re.sub(r"\]$", "", string)
     string = re.sub(r"[_\[\./]", "-", string)
     return string
+
+
+def assert_model_during_op(op, *assert_args, **assert_kwargs):
```

Comment:
Name could be more descriptive. Also one line docstring may be helpful.

Reply:
How would you name it?

Reply:
assert_model_during_operation sounds good